docs/book/src/topics/crd-changes/v1alpha7-to-v1beta1.md

```diff
@@ -14,7 +14,7 @@
       - [Change to image](#change-to-image)
```

Comment:
this section could be updated to mention the usage of `ImageParam` instead of `ImageFilter`

Reply:
Also, is there any plan to change the serverGroupFilter to become serverGroupParam?

Reply:
ARGH, I missed one! 🤦